uint8 LED_ID 
uint8[3] RGB